src/parameter.cpp

```diff
@@ -152,6 +157,11 @@ parameter::operator int64_t() const
         return static_cast<int64_t>(uintValue);
     }
 
+    static constexpr long double int64_max = std::numeric_limits<int64_t>::max();
+    if (type == DDWAF_OBJ_FLOAT && f64 <= int64_max) {
```

Comment:
needs to check `f64 >= int64_min` to avoid undefined behavior

Reply:
Yes, you're right I missed that.